docs/sources/alerting/fundamentals/alert-rules/_index.md

```diff
@@ -16,27 +21,112 @@ weight: 100
+
+The following diagram shows how data source-managed alerting works.
+
+{{< figure src="/media/docs/alerting/loki-mimir-rule.png" max-width="750px" caption="Grafana Mimir/Loki-managed alerting" >}}
+
+1. Alert rules are created and stored within the data source itself.
+1. Alert rules can only be created based on Prometheus data.
+1. Alert rule evaluation and delivery is distributed across multiple nodes for high availability and fault tolerance.
+
+### Recording rules
+
+A recording rule allows you to pre-compute frequently needed or computationally expensive expressions and save their result as a new set of time series. This is useful if you want to run alerts on aggregated data or if you have dashboards that query computationally expensive expressions repeatedly.
+
+Querying this new time series is faster, especially for dashboards since they query the same expression every time the dashboards refresh.
+
+Grafana Enterprise offers an alternative to recorded rules in the form of recorded queries that can be executed against any data source.
+
+For more information on recording rules, refer to [Create recording rules][create-recording-rules].
+
+## Comparison between alert rule types
+
+When choosing which alert rule type to use, consider the following comparison between Grafana-managed and data source-managed alert rules.
+
+{{< responsive-table >}}
+| <div style="width:200px">Feature</div> | <div style="width:200px">Grafana-managed alert rule</div> | <div style="width:200px">Data source-managed alert rule |
+| ----------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Create alert rules<wbr /> based on data from any of our supported data sources | Yes | No: You can only create alert rules that are based on Prometheus data. The data source must have the Ruler API enabled. |
+| Mix and match data sources | Yes | No |
+| Includes support for recording rules | No | Yes |
+| Add expressions to transform<wbr /> your data and set alert conditions | Yes | No |
+| Use images in alert notifications | Yes | No |
+| Scaling | More resource intensive, depend on the database, and are likely to suffer from transient errors. They only scale vertically. | Store alert rules within the data source itself and allow for “infinite” scaling. Generate and send alert notifications from the location of your data. |
+| Alert rule evaluation and delivery | Alert rule evaluation and delivery is done from within Grafana, using an external Alertmanager; or both. | Alert rule evaluation and alert delivery is distributed, meaning there is no single point of failure. |
+
+{{< /responsive-table >}}
```

Comment:
This shortcode is ineffectual now and can be removed without changing functionality.
```suggestion
| <div style="width:200px">Feature</div> | <div style="width:200px">Grafana-managed alert rule</div> | <div style="width:200px">Data source-managed alert rule |
| ----------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------- |
| Create alert rules<wbr /> based on data from any of our supported data sources | Yes | No: You can only create alert rules that are based on Prometheus data. The data source must have the Ruler API enabled. |
| Mix and match data sources | Yes | No |
| Includes support for recording rules | No | Yes |
| Add expressions to transform<wbr /> your data and set alert conditions | Yes | No |
| Use images in alert notifications | Yes | No |
| Scaling | More resource intensive, depend on the database, and are likely to suffer from transient errors. They only scale vertically. | Store alert rules within the data source itself and allow for “infinite” scaling. Generate and send alert notifications from the location of your data. |
| Alert rule evaluation and delivery | Alert rule evaluation and delivery is done from within Grafana, using an external Alertmanager; or both. | Alert rule evaluation and alert delivery is distributed, meaning there is no single point of failure. |
```